iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ReplaceTeePointByVoltageLevelOnLineBuilder.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public class ReplaceTeePointByVoltageLevelOnLineBuilder {
+
+    private String line1ZId = null;
+    private String lineZ2Id = null;
+    private String lineZPId = null;
+    private String voltageLevelId = null;
+    private String bbsOrBusId = null;
+    private String line1CId = null;
+    private String line1CName = null;
+    private String lineC2Id = null;
+    private String lineC2Name = null;
+
+    public ReplaceTeePointByVoltageLevelOnLine build() {
+        return new ReplaceTeePointByVoltageLevelOnLine(line1ZId, lineZ2Id, lineZPId, voltageLevelId, bbsOrBusId,
+                                                       line1CId, line1CName, lineC2Id, lineC2Name);
+    }
+
+    public ReplaceTeePointByVoltageLevelOnLineBuilder withLine1ZId(String line1ZId) {
+        this.line1ZId = line1ZId;
```

Comment:
Maybe you can add ```if (line1CId == null) { line1CId = line1ZId; }``` ? That way, the value by default in the same as the one used for tee point (of course, it can also be overwritten, it is only if it has been left undefined).

Reply:
Done